boards/native/Makefile.include

```diff
@@ -74,8 +74,8 @@ else
 	export PORT =
 endif
 
-ifeq (,$(filter $(PORT),$(TERMFLAGS)))
-    export TERMFLAGS += $(PORT)
+ifneq ($(PORT),$(firstword $(TERMFLAGS)))
+    export TERMFLAGS := $(PORT) $(TERMFLAGS)
```

Comment:
I don't know why this is here in the first place - just add the port unconditionally.
Specifying a `$(PORT)` via externally set `$(TERMFLAGS)` is hereby not supported anymore.